week3_model_free/sarsa.ipynb

```diff
@@ -177,7 +177,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Now we gonna implement Expected Value SARSA on top of it."
+    "Now we gonna to implement Expected Value SARSA on top of it."
```

Comment:
Yo, "gonna" has "to" in it.

It should be either "gonna implement" or "going to implement".